product/views.py

```diff
@@ -1,3 +1,43 @@
+import json
+
+from django.views     import View
+from django.http      import JsonResponse
 from django.shortcuts import render
+ 
+from .models     import Category, Subcategory, Product, ProductImage, Discount, Review
```

Comment:
```suggestion
from .models     import  (
    Category,
    Subcategory, 
    Product, 
    ProductImage, 
    Discount, 
    Review
)
```
이정도 개수면 이렇게  정렬해 주세요.